src/eko/evolution_operator/__init__.py

```diff
@@ -436,8 +436,11 @@ def run_op_integration(
                 temp_dict[label] = res[:2]
             column.append(temp_dict)
         logger.info(
-            f"{self.log_label}: computing operators: - {k+1}/{self.grid_size}"
-            f" took: {(time.perf_counter() - start_time):6f} s"
+            "%s: computing operators - %u/%u took: %6f s",
+            self.log_label,
+            k + 1,
+            self.grid_size,
+            (time.perf_counter() - start_time),
```

Comment:
Actually, why? .-.

Reply:
I can't find the blog post where I read it first, but lazy logging is actually recommanded both for performance and security reasons, because this way you ensure lazy evaluation ... it's also written here: https://medium.com/flowe-ita/logging-should-be-lazy-bc6ac9816906, https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205

Reply:
Good to know